jetty-server/src/test/java/org/eclipse/jetty/server/ConnectorTimeoutTest.java

```diff
@@ -766,8 +773,156 @@ public void testMaxIdleWithDelayedDispatch() throws Exception
         Assert.assertTrue(System.currentTimeMillis() - start > minimumTestRuntime);
         Assert.assertTrue(System.currentTimeMillis() - start < maximumTestRuntime);
     }
-    
 
+    @Test(timeout=60000)
+    public void testSlowClientRequestNoLimit() throws Exception
+    {
```

Comment:
I think these tests should be moved to a place where we can run them for both HTTP/1.1 and HTTP/2. We already have such places in module `tests`, where the tests can depend on both the `server` module and the `http2` modules.